libutils/logging.c

```diff
@@ -310,6 +311,148 @@ void Log(LogLevel level, const char *fmt, ...)
     va_end(ap);
 }
 
+
+
+static bool module_is_enabled[LOG_MOD_MAX];
+static const char *log_modules[LOG_MOD_MAX] =
+{
+    "",
+    "evalctx",
+    "expand",
+    "iterngn",
```

Comment:
I'll go for "iterations" then and "LOG_MOD_ITERATIONS". The idea was trying to keep it short, since it is supposed to be dev-only.